satellite/console/users_test.go

```diff
@@ -54,7 +54,7 @@ func TestUserRepository(t *testing.T) {
 			ID:           testrand.UUID(),
 			FullName:     name,
 			ShortName:    lastName,
-			Email:        email,
+			Email:        "new" + email,
```

Comment:
Why do we need this change?

Reply:
@stefanbenten 
because we run tests twice. First time we add user with partnerID, second without. 
One of the tests is created to count users with specified email. 
after first call of testUsers we have 2 users with "email@mail.test". after second call - 3
so i should dynamically change expected number of users in my test. 
or change email of user that will be inserted in second run of testUsers function